### PR TITLE
zh, en: update release note for v2.0.7

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Add release note for v2.0.7

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/releases/2.0.7.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+发版日期：2021 年 9 月 29 日
+
+DM 版本：2.0.7
+
+## Bug 修复
+
+- 修复 source 配置中的 `enable-gtid` 从 `false` 切换到 `true` 时，报错 binlog 被清除的问题 [#2094](https://github.com/pingcap/dm/pull/2094)
+- 修复 schema tracker 的内存泄漏问题 [#2133](https://github.com/pingcap/dm/pull/2133)
+
+## 改进之处
```
Comment (glorv) [inline zh/releases/2.0.7.md:16]: 之前的 Release Note 里面都叫「改进提升」

File: en/releases/2.0.7.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Bug fixes
+
+- Fix the error that binlog event is purged when switching `enable-gtid` in source configuration from `false` to `true` [#2094](https://github.com/pingcap/dm/pull/2094)
+- Fix the memory leak problem of schema-tracker [#2133](https://github.com/pingcap/dm/pull/2133)
+
+## Improvements
+
+- Disable background statistic job in schema tracker to reduce CPU consumption [#2065](https://github.com/pingcap/dm/pull/2065)
+- Support regular expressions rules for online DDL shadow and trash tables [#2139](https://github.com/pingcap/dm/pull/2139)
```
Comment (glorv) [inline en/releases/2.0.7.md:19]: Do we need to add document for this feature

File: zh/releases/2.0.7.md (hunk 1 above)
Comment (lichunzhu) [reply zh/releases/2.0.7.md:16]:
```suggestion
## 改进提升
```

File: en/releases/2.0.7.md (hunk 2 above)
Comment (lichunzhu) [reply en/releases/2.0.7.md:19]: Yes, I will add a document later

Comment (Liuxiaozhen12) [inline en/releases/2.0.7.md:19]: never see "shadow and trash tables" in the docs before. Are they the same as the tables of gh-ost and pt-osc? ref: https://docs.pingcap.com/zh/tidb-data-migration/stable/feature-online-ddl
```suggestion
- Support regular expressions for online DDL shadow and trash tables [#2139](https://github.com/pingcap/dm/pull/2139)
```

**Review by Liuxiaozhen12 — COMMENTED** [review]
rest LGTM

**Review by Liuxiaozhen12 — APPROVED** [review]

**Comment by glorv** [comment]
LGTM

**Review by glorv — APPROVED** [review]

**Pull request merged**